pennylane/ops/identity.py

```diff
@@ -328,6 +328,29 @@ def compute_diagonalizing_gates(
         """
         return []
 
+    @staticmethod
+    def compute_decomposition(phi, wires=None):  # pylint:disable=arguments-differ,unused-argument
+        r"""Representation of the operator as a product of other operators (static method).
```

Comment:
```suggestion
        r"""Representation of the operator as a product of other operators (static method).
        
        .. note::
        
            The ``GlobalPhase`` operation decomposes to the identity. Support for global phase
            was added in v0.33 and was ignored in earlier versions of PennyLane. Setting
            global phase to decompose to the identity allows existing devices to maintain
            current support for operations which now have ``GlobalPhase`` in the
            decomposition pipeline.
```

Reply:
should we say it "decomposes to identity"? Could be confusing because we have an operator called "Identity" and it doesn't decompose to that